### DISCUSSION TITLE
Password for chiper 2022

### DISCUSSION THREAD

**Discussion by malikshi** [shadowsocks/shadowsocks-org#197] [Q&A] [answered]
Using nightly ss rust. I can't put normal/plain password and i tried to use https://www.base64encode.org/ to create password it's seem not valid causes only 11bit. Is there any easy way to create password for new cipher 2022

**Comment by database64128** [comment, marked as answer]
For `2022-blake3-aes-128-gcm`: `openssl rand -base64 16`
For other methods: `openssl rand -base64 32`

**Reply by malikshi** [reply]
Thanks you sir.
